inspirehep/modules/forms/static/js/forms/experiments_typeahead.js

```diff
@@ -30,11 +30,11 @@ define([
     this.dataEngine = new Bloodhound({
       name: 'experiments',
       remote: {
-        url: '/api/experiments?q=experimentautocomplete:%QUERY*',
+        url: '/api/experiments/_suggest?experiment=%QUERY',
         filter: function(response) {
-          return response.hits.hits.sort(function (x, y) {
-            var x_title = x.metadata.legacy_name,
-                y_title = y.metadata.legacy_name;
+          return response.experiment[0].options.sort(function (x, y) {
```

Comment:
I am just not sure about keeping this sort, since AFAIK completion suggester returns in relevance based order.

Reply:
@harunurhan  the suggestions are retrieved in alphabetical order. It is an ES limitation. Right, @jacquerie ?

Reply:
I don't know, I remember like I have seen `_score` in the es response at some point, so I thought, order is according to that.

Reply:
@harunurhan `_score` is the new replacement keyword of `payload` in v5 I think.

Reply:
@harunurhan if i remember correctly when I checked with @jacquerie all suggestions had `_score` = 1, which did not make sense.